CHANGELOG.md

```diff
@@ -2,13 +2,121 @@
 
 ## Unreleased
 
+## v0.120.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.120.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.120.0) and the [opentelemetry-collector-contrib v0.120.1](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.120.1) releases where appropriate.
+
+### 🚩 Deprecations 🚩
+
+- (Contrib) `opencensusexporter`: Deprecate the OpenCensus exporter ([#36791](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36791))
+  Migrate to use the OTLP exporter moving forward.
+- (Contrib) `opencensusreceiver`: Deprecate the OpenCensus receiver ([#36791](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36791))
+  Migrate to use the OTLP receiver moving forward.
+
 ### 🛑 Breaking changes 🛑
 
-- (all) Bump minimum go version to 1.23 ([#5920](https://github.com/signalfx/splunk-otel-collector/pull/5920))
+- (All) Bump minimum go version to 1.23 ([#5920](https://github.com/signalfx/splunk-otel-collector/pull/5920), [#37875](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37875), [#12370](https://github.com/open-telemetry/opentelemetry-collector/pull/12370))
```

Comment:
Our changelog audience is end-users not go developers. I don't think we need this item.

Reply:
I'll remove this then. In our distro and in upstream this change was added to CHANGELOG.md and not CHANGELOG-API.md, so I included it.